go-controller/pkg/ovn/policy.go

```diff
@@ -466,38 +402,31 @@ func (oc *DefaultNetworkController) addAllowACLFromNode(nodeName string, mgmtPor
 	return nil
 }
 
-func getDefaultDenyPolicyACLName(ns string, aclT aclType) string {
-	var defaultDenySuffix string
-	switch aclT {
-	case lportIngress:
-		defaultDenySuffix = ingressDefaultDenySuffix
-	case lportEgressAfterLB:
-		defaultDenySuffix = egressDefaultDenySuffix
-	default:
-		panic(fmt.Sprintf("Unknown acl type %s", aclT))
-	}
-	return joinACLName(ns, defaultDenySuffix)
-}
-
-func getDefaultDenyPolicyExternalIDs(aclT aclType) map[string]string {
-	return map[string]string{defaultDenyPolicyTypeACLExtIdKey: string(aclTypeToPolicyType(aclT))}
-}
-
-func getARPAllowACLName(ns string) string {
-	return joinACLName(ns, arpAllowPolicySuffix)
+func (oc *DefaultNetworkController) getDefaultDenyPolicyACLIDs(ns string, aclT aclType,
```

Comment:
now that there are "aclType" and "netpolDefaultDenyACLType" its a little confusing. Looks like aclType is when acl is applied in OVN pipeline:
```
	// lportIngress will be converted to direction="to-lport" ACL
	lportIngress aclType = "to-lport"
	// lportEgressAfterLB will be converted to direction="from-lport", options={"apply-after-lb": "true"} ACL
	lportEgressAfterLB aclType = "from-lport-after-lb"
)
```

and  netpolDefaultDenyACLType is default Deny vs arp allow.

Maybe it makes sense to change aclType to aclPipelineType?